articles/service-fabric/cluster-security-certificate-management.md

```diff
@@ -25,35 +25,35 @@ As we've seen in the [companion article](cluster-security-certificates.md), a ce
+Aspects such as securing/managing domain names, enrolling into certificates, or setting up authorization controls to enforce certificate issuance are beyond the scope of this article. Refer to the Registration Authority (RA) of your favorite Public Key Infrastructure (PKI) service. Microsoft-internal consumers: please reach out to Azure Security.
 
 ## Roles and entities involved in certificate management
 The security approach in a Service Fabric cluster is a case of "cluster owner declares it, Service Fabric runtime enforces it". By that we mean that almost none of the certificates, keys, or other credentials of identities participating in a cluster's functioning come from the service itself; they are all declared by the cluster owner. Furthermore, the cluster owner is also responsible for provisioning the certificates into the cluster, renewing them as needed, and ensuring the security of the certificates at all times. More specifically, the cluster owner must ensure that:
-  - certificates declared in the NodeType section of the cluster manifest can be found on each node of that type, according to the [presentation rules](cluster-security-certificates.md#presentation-rules)
-  - certificates declared above are installed inclusive of their corresponding private keys
-  - certificates declared in the presentation rules should pass the [validation rules](cluster-security-certificates.md#validation-rules) 
+  - Certificates declared in the NodeType section of the cluster manifest can be found on each node of that type, according to the [presentation rules](cluster-security-certificates.md#presentation-rules)
+  - Certificates declared above are installed with their corresponding private keys included.
+  - Certificates declared in the presentation rules should pass the [validation rules](cluster-security-certificates.md#validation-rules) 
 
-Service Fabric, for its part, assumes the responsibilities of:
-  - locating/finding certificates matching the declarations in the cluster definition  
-  - granting access to the corresponding private keys to Service Fabric-controlled entities on a 'need' basis
-  - validating certificates in strict accordance with established security best-practices and the cluster definition
-  - raising alerts on impending expiration of certificates, or failures to perform the basic steps of certificate validation
-  - validating (to some degree) that the certificate-related aspects of the cluster definition are met by the underlying configuration of the hosts 
+Service Fabric, for its part, assumes the following responsibilities:
+  - Locating certificates matching those declared in the cluster definition
```

Comment:
>those declared [](http://example.com/codeflow?start=35&length=14)

The original "the declarations" was intended here. This line refers to finding a certificate according to a declaration, e.g.: FindType=FindByCommonName FindValue="blah.com"

If you consider that "those declared" is still correct/explains the meaning of the phrase clearly, then we can keep it, otherwise I would suggest reversing or rephrasing this line.

Reply:
That`s a good point, thanks. I've rephrased it, hope it's more clear now.

Reply:
Yup, thanks!